### ISSUE TITLE
New Official Stats tracker support

### ISSUE THREAD

**Issue by BadCoder1337** [danielwerg/r6api.js#42]
![image](https://user-images.githubusercontent.com/16510197/96872365-35708980-147c-11eb-9f25-1e11e03a7f2b.png)
OMG IT'S HAPPENED!!!
There's NEW map stats, stats drilldown by time and more!
R6Tab already implemented new site parsing.

**Comment by BadCoder1337** [comment]
THERE'S ALSO SEASONAL DATA FROM BLACK ICE

**Comment by danielwerg** [comment]
Yeah saw it yesterday, played around with new endpoints will try to implement them in typescript rewrite, gonna try to push new branch today

**Comment by danielwerg** [comment]
> gonna try to push new branch today

well

I was hoping they would fix few bugs in a following week after they pushed new site before I would try to integrate in into wrapper but it's almost been two weeks...

A lot of things are still broken, for example, amount of matches per seasons is incorrect it's doesn't include abandons only wins and losses. And it's also weird how they now use two endpoints which has same data such as matches won, matches lost for "SEASONS" tab.

I was hoping we would finally get proper API but instead of that now it's even bigger mess with new endpoints. Not sure how I will implement it yet, there A LOT of new stuff.

**Comment by danielwerg** [comment]
Okay, I'm trying to justify spending time and adding support for this. So far don't think it is worth it. There still some things possibly broken (ex: https://i.imgur.com/a79DWru.png). Amount of kills/deaths and matches played are different for some reason.

![](https://i.imgur.com/a79DWru.png)

@BadCoder1337 you mentioned r6tab implemented new api, but I can't really see where they used it.

Also didn't someone else implemented new api somewhere like api wrapper for other languages or tracker site?

**Comment by BadCoder1337** [comment]
R6Tab collects map statistics, which doesn't present in r6api.js. 

Currently there's a huge backlog between open source wrappers and R6Tab. They and single twitter bot got access to BattlEye feed. They  somehow detect player squads in the browser.

Somebody from the Modding community told that Ubisoft spoon-fed API docs to R6Tab programmers. I don't know the truth, I discovered the endpoint for player inventory and a few profile related, but haven't any clue for further exploration.

**Comment by danielwerg** [comment]
I'm pretty sure all things you listed they are getting from their overwolf app. If you go to official tracker and r6tab and compare your stats for specific map you will see.

**Comment by danielwerg** [comment]
### Here is comparison:
![](https://i.imgur.com/Iw2RTzN.png)

**Comment by danielwerg** [comment]
`r6s-stats.ubisoft.com` is replaced by `prod.datadev.ubisoft.com` on ubi site
